phpBB/phpbb/install/helper/container_factory.php

```diff
@@ -157,6 +181,23 @@ protected function build_container()
 		}
 
 		// Get compatibilty globals
-		require ($this->phpbb_root_path . 'includes/compatibility_globals.' . $this->php_ext);
+		if (file_exists($this->phpbb_root_path . 'install/update/new/includes/compatibility_globals.' . $this->php_ext))
```

Comment:
couldn't you fetch and use the update_helper here? (may be related to my comment in the update_helper)